static/app/views/userFeedback/index.tsx

```diff
@@ -18,50 +18,51 @@ import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionT
 import Pagination from 'sentry/components/pagination';
 import Panel from 'sentry/components/panels/panel';
 import {SegmentedControl} from 'sentry/components/segmentedControl';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {Tooltip} from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {Organization, UserReport} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 import withOrganization from 'sentry/utils/withOrganization';
-import type {AsyncViewState} from 'sentry/views/deprecatedAsyncView';
-import DeprecatedAsyncView from 'sentry/views/deprecatedAsyncView';
 
 import {UserFeedbackEmpty} from './userFeedbackEmpty';
 import {getQuery} from './utils';
 
-interface State extends AsyncViewState {
-  reportList: UserReport[];
-}
-
 interface Props extends RouteComponentProps<{}, {}> {
   organization: Organization;
 }
 
-class OrganizationUserFeedback extends DeprecatedAsyncView<Props, State> {
-  getEndpoints(): ReturnType<DeprecatedAsyncView['getEndpoints']> {
-    const {
-      organization,
-      location: {search},
-    } = this.props;
-
-    return [
-      [
-        'reportList',
-        `/organizations/${organization.slug}/user-feedback/`,
-        {
-          query: getQuery(search),
-        },
-      ],
-    ];
-  }
-
-  getTitle() {
-    return `${t('User Feedback')} - ${this.props.organization.slug}`;
-  }
-
-  get projectIds() {
-    const {project} = this.props.location.query;
+function OrganizationUserFeedback({
+  organization,
```

Comment:
nit: with the FC you can remove `withOrganization` and use `useOrganization()` instead